lib/senders/templates/verify_primary.txt

```diff
@@ -0,0 +1,12 @@
+{{t "Verify primary email"}}
```

Comment:
Looking at the other text templates, they don't seem to include the title (presumably because it doesn't look like a title unformatted). Should we delete this?

Reply:
Actually, some of them do. Maybe it will look better if it's followed by a blank line rather than more text.